blocks/api/raw-handling/utils.js

```diff
@@ -110,8 +115,21 @@ export function isClassWhitelisted( tag, name ) {
 	);
 }
 
+/**
+ * Whether or not the given node is embedded content.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Embedded_content
+ *
+ * @param {Node} node The node to check.
+ *
+ * @return {boolean} True if embedded content, false if not.
+ */
+export function isEmbedded( node ) {
+	return !! embeddedWhiteList.hasOwnProperty( node.nodeName.toLowerCase() );
```

Comment:
The type coercion is unnecessary (`hasOwnProperty` will certainly return a boolean).